lightpath/controller.py

```diff
@@ -42,115 +49,402 @@ def __init__(self, client, endstations=None):
         self.client = client
         self.containers = list()
         self.beamlines = dict()
+        self.graph = nx.DiGraph()
```

Comment:
This empty graph is overwritten on line 95 when we load the facility. Should this be a type annotation instead?